website/static/js/osfHelpers.js

```diff
@@ -390,6 +390,35 @@ var htmlEscape = function(text) {
     return $('<div/>').text(text).html();
 };
 
+/**
+ * Resize table to match thead and tbody column
+ */
+
+var tableResize = function(selector, checker) {
```

Comment:
Ping @caneruguz for a 2nd review of this. Are there alternative solutions to this problem?